components/feature/webcompat-reporter/src/main/assets/extensions/webcompat-reporter/background.js

```diff
@@ -117,18 +127,21 @@ function getWebCompatInfoForTab(tab) {
         delete graphicsPrefs["layers.acceleration.force-enabled"];
       }
 
-      return Object.assign({}, {
-        tabId: id,
-        blockList,
-        details: Object.assign(graphicsPrefs, {
-          buildID,
-          channel,
-          frameworks,
-          GPUs,
-          hasTouchScreen,
-        }),
-        url,
-      });
+      return Object.assign(
```

Comment:
This was `prettier` from inside mozilla-central doing its thing. I could manually undo this, but since this will happen inside m-c anyway, it's probably a good thing to have that here as well.